### PR TITLE
Add juliahchen San Diego file

### LINKED ISSUE #31: Create a new file for juliahchen San Diego, CA
- [x] Create a branch
- [x] Add Commits
- [x] Open a PR
- [ ] Have a discussion
- [ ] Seek Approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by juliahchen**
closes #31

### CODE REVIEW

File: juliahchen-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# San Diego, CA
+
+### Food Recommendations
+- Fish District in Carlsbad
+- Karl Strauss in Carlsbad
+
+### Places to Go
+- Safari Park
```
Comment (davidmcgimpsey) [inline juliahchen-sandiego.md:9]: Sounds good - What's the best time to visit.

**Comment by davidmcgimpsey** [comment]
Some conversation level comments

**Review by davidmcgimpsey — APPROVED** [review]
Approved

File: juliahchen-sandiego.md (hunk 1 above)
Comment (juliahchen) [reply juliahchen-sandiego.md:9]: Any time really spring is a great time of year

**Pull request merged**